README.md

```diff
@@ -149,28 +153,28 @@ type  │                    data
      format
 ```
 
-### Encrypted data formats
+### 5. encrypted formats
 
 When content is encrypted (in other words, "boxed") in SSB, it is provided as
 uninterpretable bytes, plus a tag that identifies which algorithm was used for
 encrypting it, such as `box` or `box2`.
 
 | Type code | Format code | Data length | Format name | Specification   |
-|-----------|-------------|-------------|-------------|-----------------|
-| 5         | 0           | Arbitrary   | box         | [private box]   |
+|:---------:|:-----------:|-------------|-------------|-----------------|
+| 5         | 0           | Arbitrary   | box1        | [private box]   |
 | 5         | 1           | Arbitrary   | box2        | [envelope spec] |
 
-### Generic data formats
+### 6. generic formats
 
 BFE supports encoding data types with no semantics attached to them. They are
 merely categorized into formats that represent their data type.
 
 | Type code | Format code | Data length | Format name | Specification                 |
-|-----------|-------------|-------------|-------------|-------------------------------|
-| 6         | 0           | Arbitrary   | UTF8 string | [UTF8]                        |
-| 6         | 1           | 1 byte      | Boolean     | Data byte is 0 for False, 1 for True |
-| 6         | 2           | 0 bytes     | Nil         | [null pointer]                |
-| 6         | 3           | Arbitrary   | Arbitrary bytes | N/A |
+|:---------:|:-----------:|-------------|-------------|-------------------------------|
+| 6         | 0           | Arbitrary   | string-UTF8 | [UTF8]                        |
```

Comment:
Would appreciate if this is reverted.

Reply:
Please respond to my values /proposals
Specifically, I don't want spaces in the name because it makes uris have to have some rule for dealing with them (or you could propose one)

Reply:
The response to values/proposals is that this markdown document is for humans to read, so let's use formal English, while the json is for computers to parse. In the markdown we should be trying to explain to people what this is.

As for URIs, we're not going to have SSB URIs for these Generic data formats, so I don't understand what's the need for the hyphen.

Reply:
Agree about ssb uri for generic types. I reckon that can be reverted happily then